packages/application/src/shell.ts

```diff
@@ -2231,18 +2266,15 @@ namespace Private {
       this.node.removeEventListener('click', this);
       super.onBeforeDetach(msg);
     }
-
-    private _focusLeftSideBar() {
-      this._shell.expandLeft();
-    }
     private _shell: ILabShell;
 
-    private _createSkipLink(skipLinkText: string): void {
+    private _createSkipLink(skipLinkText: string, area: ILabShell.Area): void {
       const skipLink = document.createElement('a');
       skipLink.href = '#';
       skipLink.tabIndex = 1;
```

Comment:
```suggestion
      skipLink.tabIndex = 0;
```

it is recommended to only use `0` and `-1` for `tabindex`.